src/gui/widgets/multiline_text.hpp

```diff
@@ -132,16 +132,13 @@ class multiline_text : public text_box_base
 		update_layout();
 	}
 
-	/** Inherited from text_box_base. */
-	void set_cursor(const std::size_t offset, const bool select, const bool autoscroll = true)
+	/** Inherited from text_box_base, defaults the autoscroll argument to true. */
```

Comment:
This "defaults the autoscroll argument" is outdated

Reply:
Updated comment.